examples/sc.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: storage.k8s.io/v1beta1
+kind: StorageClass
+metadata:
+  name: rbd-csi-provisioner 
+provisioner: csi-rbd
+parameters:
+   monitors: 172.24.0.6:6789
+   pool: rbd
+   adminID: admin
+   adminSecret: AQAmsGBap4EoBhAAET/Hc7fBqAZj/cy7cDcoQA==
```

Comment:
Limitation will be addressed, refer to https://github.com/kubernetes-csi/external-provisioner/issues/28